make/modules/jdk.hotspot.agent/Lib.gmk

```diff
@@ -54,6 +59,9 @@ ifeq ($(call isTargetOs, linux), true)
 else
   LIBSAPROC_LINK_TYPE := C
 endif
+ifeq ($(call isTargetOs, bsd), true)
+  SA_TOOLCHAIN := TOOLCHAIN_LINK_CXX
```

Comment:
This is not how it is done anymore. You need to update the check where LIBSAPROC_LINK_TYPE is determined.

Reply:
Thanks, that's noted.